contracts/cw20-gov/src/contract.rs

```diff
@@ -0,0 +1,1996 @@
+#[cfg(not(feature = "library"))]
+use cosmwasm_std::entry_point;
+use cosmwasm_std::{
+    to_binary, Binary, Deps, DepsMut, Env, MessageInfo, Response, StdError, StdResult, Uint128,
+};
+
+use cw2::set_contract_version;
+use cw20::{
+    BalanceResponse, Cw20Coin, Cw20ReceiveMsg, DownloadLogoResponse, EmbeddedLogo, Logo, LogoInfo,
+    MarketingInfoResponse, MinterResponse, TokenInfoResponse,
+};
+
+use crate::allowances::{
+    execute_burn_from, execute_decrease_allowance, execute_increase_allowance, execute_send_from,
+    execute_transfer_from, query_allowance,
+};
```

Comment:
You may want to consider just inheriting these from `cw20-base`. Could delete the allowances file then. Probably more things we could inherit from that contract since we only made a small change to the `BALANCES` state.

Reply:
yeah I think that is a really good idea for endpoints that do not interact with balance.